articles/virtual-machines/virtual-machines-linux-extensions-customscript.md

```diff
@@ -136,8 +136,8 @@ The Azure Custom Script Extension can be run at Virtual Machine deployment time
 
 ```json
 {
-    "name": "scriptextensiondemo",
-    "type": "extensions",
+    "name": "[concat(parameters('scriptextensiondemoName'), 'scriptextensiondemo')]",
+    "type": "Microsoft.Compute/virtualMachines/extensions",
```

Comment:
This is complicate. As I understand it, a script extension can be nested under a VM / VMSS, or at the 'root' of the template. If nested (more likely configuration), this does not appear to work.

Can you provide a sample / source material for this?